jwst/saturation/tests/test_saturation.py

```diff
@@ -213,11 +213,11 @@ def test_nans_in_mask(setup_nrc_cube):
     correct_for_NaN(satmap.data, satmap.dq)
     output = do_correction(data, satmap)
 
-    # Check that NaN reference value gets reset to HUGE_NUM
+    # Check that NaN reference value gets reset to ATOD_LIMIT
     # Check that reference DQ is set to NO_SAT_CHECK
     # Check that output GROUPDQ is not flagged as saturated
     # Check that output PIXELDQ is set to NO_SAT_CHECK
-    assert satmap.data[500, 500] == huge_num
+    assert satmap.data[500, 500] == ATOD_LIMIT
     assert satmap.dq[500, 500] == dqflags.pixel['NO_SAT_CHECK']
```

Comment:
May want to have a test where one of the input DQ arrays has another bit set and then makes sure output doesn't blow away that bit.  See my comments above.

Reply:
Done